modules/delegates/api/fetchGithubDelegates.ts

```diff
@@ -13,7 +13,7 @@ async function extractGithubInformation(
   folder: GithubPage
 ): Promise<DelegateRepoInformation | undefined> {
   try {
-    const folderContents = await fetchGitHubPage(owner, repo, folder.path);
+    const folderContents = await fetchGitHubPage(owner, repo, folder.path, 1);
```

Comment:
Nice, think we could put the ID's into an enum? Might be easier to read if it's something like `GithubKeys.Delegates` or something similar.

Reply:
Sounds good 👍

Reply:
Good suggestion, I added the enum, and realized we should just use 3 different tokens, there's two different functions where we fetch the delegates folder, so just using the same token for both of those @b-pmcg